ci/build-docker.sh

```diff
@@ -179,7 +179,7 @@ cd "${BUILD_DIR}"
 ctest --output-on-failure
 
 # Run the integration tests.
-for subdir in google/cloud/bigtable storage; do
+for subdir in google/cloud/bigtable google/cloud/storage; do
```

Comment:
What about:

```sh
for subdir in google/cloud/* ; do
```

Reply:
I guess `firestore` won't work yet? In which case, this is fine, and you can do something like:

```sh
for subdir in google/cloud/{bigtable,storage} ; do 
```

Thoughts?

Reply:
Done.